### PR TITLE
pester: generate report in HTML

### PR DESCRIPTION
**Pull request by sodar**
Blocked by: https://github.com/Juniper/contrail-windows-ci/pull/1

### CODE REVIEW

**Comment by sodar** [comment]
recheck no bug

**Comment by sodar** [comment]
recheck no bug

File: Jenkinsfile [hunk 1]
```diff
@@ -188,6 +188,19 @@ pipeline {
 
     post {
         always {
+            node('tester') {
+                deleteDir()
+                unstash 'CIScripts'
+                script {
+                    try {
+                        unstash 'testReport'
+                        powershell script: './CIScripts/GenerateTestReport.ps1'
```
Comment (jacek-izykowski) [inline Jenkinsfile]: What do you think about passing report file name(s) as a parameter(s) to GenerateTestReport?

Comment (sodar) [reply Jenkinsfile]: I think it might be a good addition. We'll change when reports from othere testing frameworks will occur.

**Pull request merged**